src/olympia/editors/templates/editors/review.html

```diff
@@ -168,12 +168,15 @@ <h3 id="history">
         {{ _("You can still submit this form, however only do so if you know it won't conflict.") }}
       </div>
 
-      <label for="id_comments">{{ form.comments.label }}</label>
-      {{ form.comments }}
-      {{ form.comments.errors }}
-      <div class="review-actions-canned">
-        {{ _('Insert canned response...') }}
-        {{ form.canned_response }}
+      <div class="review-actions-section data-toggle"
+           data-value="{{ actions_comments|join("|") }}">
```

Comment:
Hiding/showing the comments depending on the actions - it's done by `static/js/zamboni/editors.js` if `data-toggle` class is present, according to the actions in `data-value`.